src/notification/AzureServiceBus.js

```diff
@@ -0,0 +1,49 @@
+import { ServiceBusClient } from '@azure/service-bus';
+
+/**
```

Comment:
Do we need this empty comment, or should that say something?

Reply:
Addressed